client/index.js

```diff
@@ -141,6 +154,24 @@ function getImagePath (key) {
   if (key === '.') key = 'dot'
   if (key === '/') key = 'slash'
   if (key === '\\') key = 'backslash'
+
+  //Numpad -- make this for no repeat images
+  if (key === 'num-/') key = 'slash'
```

Comment:
I think can be done simpler by just doing:

```js
if (key.indexOf('num-') === 0) {
  key = key.substring(4)
}
```

Then you need to also add 'plus' and 'star' to the if statements above.